bup_cron/__init__.py

```diff
@@ -1074,12 +1085,12 @@ def main():
         if not os.path.exists(os.environ['BUP_DIR']):
             if not Bup.init(args.remote):
                 bail(3, timer, 'failed to initialize bup repo')
-        else:
+
+        with Pidfile(args.pidfile):
             if args.clear:
                 if not Bup.clear_index():
```

Comment:
Addressed in https://github.com/blueyed/bup-cron/commit/6f0cde147f3253c33108b1b914011f3a5ae727cb.